MapboxNavigationTests/InstructionsCardSnapshotTests.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+import XCTest
+import SnappyShrimp
+import MapboxDirections
+@testable import TestHelper
+@testable import MapboxNavigation
+@testable import MapboxCoreNavigation
+
+class InstructionsCardSnapshotTest: SnapshotTest {
+    
+    override func setUp() {
+        super.setUp()
+        recordMode = false
+    }
+    
+    lazy var initialRoute: Route = {
+        return Fixture.route(from: jsonFileName)
+    }()
+    
+    @available(iOS 11.0, *)
+    func testInstructionsCardCollection() {
+        let host = UIViewController(nibName: nil, bundle: nil)
+        let container = UIView.forAutoLayout()
+        let subject = InstructionsCardCollection(nibName: nil, bundle: nil)
+        
+        host.view.addSubview(container)
+        constrain(container, to: host.view)
+        
+        embed(parent: host, child: subject, in: container) { (parent, guidanceCard) -> [NSLayoutConstraint] in
+            guidanceCard.view.translatesAutoresizingMaskIntoConstraints = false
+            return guidanceCard.view.constraintsForPinning(to: container)
+        }
+        
+        let fakeRoute = Fixture.route(from: "route-with-banner-instructions")
+        
+        let service = MapboxNavigationService(route: initialRoute, directions: DirectionsSpy(accessToken: "knut"), simulating: .never)
+        let routeProgress = RouteProgress(route: fakeRoute)
+        let intersectionLocation = routeProgress.route.legs.first!.steps.first!.intersections!.first!.location
+        let fakeLocation = CLLocation(latitude: intersectionLocation.latitude, longitude: intersectionLocation.longitude)
+        subject.routeProgress = routeProgress
+        subject.navigationService(service, didUpdate: routeProgress, with: fakeLocation, rawLocation: fakeLocation)
+        
+        /* TODO: Update and Re-generate for generic instructions cards and complex maneuver instruction cards */
+//        /// Validate the visible collection view cell
```

Comment:
Commented-out code.

Reply:
Yep - will be uncommented along with the `WIP` tag.

Reply:
Groovy.